test/k8sT/Services.go

```diff
@@ -2760,6 +2760,26 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 					testNodePort(true, false, false, 0) // no need to test from outside, as testDSR did it
 				})
 
+				It("Should not drop SCTP packets", func() {
+					vm1 := helpers.InitRuntimeHelper(helpers.K8s1VMName(), logger)
+					vm2 := helpers.InitRuntimeHelper(helpers.K8s2VMName(), logger)
+					ctx, cancel := context.WithCancel(context.Background())
+					defer cancel()
+
+					By("Running iperf3 server")
+					_ = vm1.ExecInBackground(ctx, "iperf3 -s")
```

Comment:
Won't this fail on GKE, EKS & co.? Maybe it would be best to use a host-network pod for the test?